lisa/sut_orchestrator/azure/features.py

```diff
@@ -80,3 +80,17 @@ def _is_supported(self) -> bool:
         if not isinstance(self._node.os, supported_distro):
             return False
         return True
+
+
+class Sriov(AzureFeatureMixin, features.Sriov):
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        super()._initialize(*args, **kwargs)
+        self._initialize_information(self._node)
+
+    def _is_supported(self) -> bool:
```

Comment:
As discussed, the distro check happens on deployment, and no other reasons for dynamic support check. so it doesn't need `_is_supported` now.